public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -384,7 +386,13 @@ export class GeomapPanel extends Component<Props, State> {
       <>
         <Global styles={this.globalCSS} />
         <div className={styles.wrap} onMouseLeave={this.clearTooltip}>
-          <div className={styles.map} ref={this.initMapRef}></div>
+          <section
+            className={styles.map}
+            tabIndex={0}
```

Comment:
This lint issue can be ignored, as keyboard handlers are being added with the help of the ref.